src/main/java/com/rabbitmq/perf/MulticastParams.java

```diff
@@ -28,6 +28,8 @@
     private long confirm = -1;
     private int consumerCount = 1;
     private int producerCount = 1;
+    private int consumerChannels = 1;
```

Comment:
We ended up with `producer/consumerChannelCount` in https://github.com/rabbitmq/rabbitmq-perf-test/commit/624e6caddf449a3d56b509b62fa545d8e5155c37. This is enough for consistency sake, the options descriptions being descriptive.